tests/components/laundrify/test_config_flow.py

```diff
@@ -0,0 +1,193 @@
+"""Test the laundrify config flow."""
+from unittest.mock import patch
+
+from laundrify_aio import exceptions
+
+from homeassistant.components.laundrify.const import DOMAIN
+from homeassistant.config_entries import SOURCE_REAUTH, SOURCE_USER, ConfigEntryState
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_CODE, CONF_SOURCE
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from . import (
+    _patch_laundrify_exchange_code,
+    _patch_laundrify_get_account_id,
+    _patch_laundrify_get_machines,
+    _patch_laundrify_validate_token,
```

Comment:
rather than have functions for these, you should define fixtures in conftest.py and include the fixture name as a parameter in your test.

Here's an example of a fixture that patches several functions: https://github.com/home-assistant/core/blob/dev/tests/components/tomorrowio/conftest.py#L30-L46

And here's an example of how you can have the test use the fixture: https://github.com/home-assistant/core/blob/dev/tests/components/tomorrowio/test_config_flow.py#L238-L240

You can also specify that a fixture should always be used for all tests, and then you don't need to do the above, by setting autouse=True. Example: https://github.com/home-assistant/core/blob/dev/tests/components/tomorrowio/conftest.py#L10-L17

Reply:
Fixed in e54dbf5bf2c4d9def50dfb20a89c9f35a78e9622